src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp

```diff
@@ -200,6 +200,7 @@ std::tuple<std::size_t, bool> PerformanceImplicitGemmBwdDataV4R1Xdlops::Calculat
 
         int GemmABlockCopyClusterLengths_GemmM  = 0;
         int GemmABlockCopyDescDataPerWriteGemmM = 0;
+        int GemmKPack                           = GetEPackLength(ctx, true);
```

Comment:
Lets GemmKPack configurable. So you will be obtained GemmKPack from config.GemmKPACK, not GetEPackLength.

Reply:
OK.